tests/cypress/e2e/features/shortcuts.js

```diff
@@ -0,0 +1,244 @@
+            cy.get('.cvat-shortcuts-settings-search input').clear();
+        }
+        cy.get('.cvat-shortcuts-settings-search input').type(randomSearchItem);
+        cy.get(searchItemClass).should('not.exist');
+        cy.get('.cvat-shortcuts-settings-search input').clear();
+        cy.get('.cvat-shortcuts-settings-search input').blur();
+    }
+
+    function registerF2F3(shouldExist) {
+        cy.get('.ant-list-item').should('exist').and('be.visible');
+        cy.get('.ant-list-item .ant-select').first().click();
+        cy.realPress(['F2']);
+        cy.wait(200);
+        cy.realPress(['F3']);
+        cy.get('.ant-modal-content').contains('Conflicting shortcuts detected');
+        cy.get(
+            shouldExist ?
+                '.ant-modal-content .ant-modal-confirm-btns .ant-btn-primary' :
+                '.ant-modal-content .ant-modal-confirm-btns .ant-btn-default',
+        ).click();
+        cy.get('.ant-list-item .ant-select').first().within(() => {
+            cy.get('.ant-select-selection-overflow-item').contains('f2 f3').should(shouldExist ? 'exist' : 'not.exist');
+        });
+    }
+
+    function saveSettings() {
+        cy.get('.cvat-settings-modal .ant-modal-footer .ant-btn-primary').click();
+    }
+
+    function checkShortcutsMounted(label) {
+        cy.get('.cvat-shortcuts-modal-window-table').should('exist').and('be.visible');
+        for (let i = 1; i < 4; i++) {
+            cy.get('.cvat-shortcuts-modal-window-table').contains(label(i));
+        }
+    }
+
+    after(() => {
+        cy.logout();
+    });
+
+    describe('Searching for a shortcut', () => {
+        it('Searching according to description', () => {
+            cy.openSettings();
+            cy.contains('Shortcuts').click();
+            cy.get('.cvat-shortcuts-settings-search input').should('exist').and('be.visible');
+            testSearchShortcuts(true);
+        });
+        it('Searching according to title', () => {
+            testSearchShortcuts(false);
+        });
+    });
+
+    describe('Registration and testing of new shortcuts', () => {
+        it('Registering a new shortcut', () => {
+            cy.get('.cvat-shortcuts-settings-collapse').should('exist').and('be.visible');
+            cy.get('.ant-collapse-header').first().click();
+            registerF2F3(false);
+            registerF2F3(true);
+        });
+        it('Registering a combination shortcut and testing if it works or not', () => {
+            cy.get('.ant-list-item').should('exist').and('be.visible');
+            cy.get('.ant-list-item .ant-select').first().click();
+            cy.realPress(['Control', 'Space']);
+            cy.wait(1050);
+            cy.closeSettings();
+            cy.realPress(['Control', 'Space']);
+            cy.get('.cvat-shortcuts-modal-window').should('exist').and('be.visible');
+            cy.wait(150);
+            cy.realPress(['Control', 'Space']);
```

Comment:
Probably it shall be combined another way.

I see three test cases: 
1. Register shortcut, no conflict, check it works
2. Register shortcut, conflict, press NO, shortcut was not set
3. Register shortcut, conflict, press YES, shortcut was set, check it works